### ISSUE TITLE
Line 87 typo

### ISSUE THREAD

**Issue by BenDrewry** [vandit15/Class-balanced-loss-pytorch#10]
original:
 cb_loss = F.binary_cross_entropy_with_logits(input = logits,target = labels_one_hot, weights = weights)

update:
 cb_loss = F.binary_cross_entropy_with_logits(input = logits,target = labels_one_hot, weight = weights)

- pytorch optimizer takes keyword 'weight', not 'weights'

Thanks for the resource, it worked great!!

**Comment by Python-Eric** [comment]
wo ! useful , it did work !

**Comment by fcakyon** [comment]
@BenDrewry @Python-Eric  feel free to use this improved/fixed/battle-tested version, includes this fix ad much more: https://github.com/fcakyon/balanced-loss#improvements
